src/ssl.c

```diff
@@ -20863,6 +20866,17 @@ unsigned long wolfSSL_X509_NAME_hash(WOLFSSL_X509_NAME* name)
         WOLFSSL_MSG("nothing to hash in WOLFSSL_X509_NAME");
         return 0;
     }
+  
+    size = wolfSSL_i2d_X509_NAME_canon(name, (byte**)&canon_name);
+    
+    if (size <= 0){
+        WOLFSSL_MSG("wolfSSL_i2d_X509_NAME_canon error");
+        return 0;
+    }
+    
+    name->name = canon_name;
```

Comment:
It looks like the pointer previously at `name->name` is being leaked?

Reply:
It was. No longer overwriting the pointer.